cropharvest/eo/eo.py

```diff
@@ -38,7 +39,20 @@
 
 
 class EarthEngineExporter:
-    def __init__(self, data_folder: Path = DATAFOLDER_PATH) -> None:
+    """
+    Export satellite data from Earth engine. It's called using the following
+    script:
+    ```
+    from cropharvest.eo import EarthEngineExporter
+
+    exporter = EarthEngineExporter()
+    exporter.export_for_labels()
```

Comment:
nit: why not just?
```python
EarthEngineExporter().export_for_labels()
```

Reply:
Yes; this makes sense. I added an issue, since it will require some changes to the Exporter class: https://github.com/nasaharvest/cropharvest/issues/57